src/env.cr

```diff
@@ -52,6 +52,29 @@ module ENV
     !!LibC.getenv(key)
   end
 
+  # Fetch an environment variable corresponding to a key. Throws an exception
+  # if the key does not exist. If a second argument or a block is provided then
+  # this value is returned instead of throwing the exception.
+  #
+  def self.fetch(key)
+    fetch(key) do
+      if block = @@block
```

Comment:
I dunno :) I took this from hash.cr, where it's `@block`. The compiler made me change this to `@@block`. I thought that `@block` was just magic to get access to the block and that `@@block` was the equivalent for classes, but thinking about it now it's probably just something that was saved off to an instance variable elsewhere. The tests pass but I guess that's because it's going through line 75. I'll look at hash.cr to understand better where that variable came from.